### PR TITLE
Feature/39515

### PR DESCRIPTION
**Pull request by UrvashiChaudhary**
Prerequisites
Create a new branch for this task according to guidelines.

Latest Stable Node Version Installed on System

Task:
Create a new folder inside extra named as 'patterns'.
Create a file for each below the pattern and also write the script for the same.

**Patterns are:**
- Diamond pattern
- Equilateral triangle

**Commands to be used must be:**
- node extra/patterns/Diamond.js
- node extra/patterns/Triangle.js

All the files must accept a command line argument containing an integer between 2 to 10.

**Acceptance Criteria:**
- The function should accept no of rows as an argument which will be provided when the function will be called.
- A pattern must have the same number of rows as provided in the function call.

### CODE REVIEW

File: extra/Diamond.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+function diamondPattern(rows){
```
Comment (adilsuccessive) [inline extra/Diamond.js]: Diamond.js should be inside extra/patterns.

File: extra/Triangle.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+function equilateralTriangle(rows){
```
Comment (adilsuccessive) [inline extra/Triangle.js]: Traingle.js should also be inside extra/patterns.

**Review by adilsuccessive — CHANGES_REQUESTED** [review]

File: extra/Triangle.js (hunk 2 above)
Comment (UrvashiChaudhary) [reply extra/Triangle.js]: Done sir, now Triangle.js is inside extra/patterns.

File: extra/Diamond.js (hunk 1 above)
Comment (UrvashiChaudhary) [reply extra/Diamond.js]: Done sir, now Diamond.js is inside extra/patterns.

**Pull request merged**